src/components/Footer.jsx

```diff
@@ -101,15 +49,48 @@ class Footer extends Component {
 
         <div className="footer_nav">
           <div className="footer-nav-link">
-            <AboutTooltip title={<AboutCard />}>
-              <p>About</p>
-            </AboutTooltip>
+            <p
+              onClick={() =>
+                this.setState({ showAbout: !this.state.showAbout })
+              }
+            >
+              About
+            </p>
+          </div>
+          <div className="footer-nav-link">Start a network</div>
+          <div className="footer-nav-link">CAMS</div>
+          <div className="footer-nav-link">
+            <React.Fragment>
```

Comment:
I think this `Fragment` is redundant. The same also applies below

Reply:
Okay, taken care of too